oci/runtime_v2.go

```diff
@@ -28,7 +57,136 @@ func (r *RuntimeV2) Version() (string, error) {
 }
 
 // CreateContainer creates a container.
-func (r *RuntimeV2) CreateContainer(c *Container, cgroupParent string) (err error) {
+func (r *RuntimeV2) CreateContainer(c *Container, cgroupParent string) error {
+	logrus.Debug("Runtimev2.CreateContainer() start")
+	defer logrus.Debug("RuntimeV2.CreateContainer() end")
+
+	// First thing, we need to start the runtime daemon
+	if err := r.startRuntimeDaemon(c); err != nil {
+		return err
+	}
+
+	// Create IO fifos
+	containerIO, err := cio.NewContainerIO(c.ID(),
```

Comment:
Do we need to close the containerIO if failed to create container?

Reply:
We do!

Reply:
done